### PR TITLE
solved task 3 calculate rectangular triangle

### PR DESCRIPTION
**Pull request by neb5562**

### CODE REVIEW

File: 3 - Rectangular triangle/solution_3.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+    # check if triangle is isosceles
+    if is_isosceles?
+      puts "Triangle is #{'also ' if is_pythagorean? && is_isosceles?}isosceles.\n"
+    end
+  end
+
+  # check if triangle is valid
+  def is_triangle?
+    ((@first_side + @second_side) > @third_side && (@second_side + @third_side) > @first_side && (@third_side + @first_side) > @second_side)
```
Comment (alexshvedko) [inline 3 - Rectangular triangle/solution_3.rb]: very long line. https://rubystyle.guide/#max-line-length

File: 3 - Rectangular triangle/solution_3.rb [hunk 2]
```diff
@@ -18,7 +18,11 @@ def calculate_rectangular_triangle
 
   # check if triangle is valid
   def is_triangle?
-    ((@first_side + @second_side) > @third_side && (@second_side + @third_side) > @first_side && (@third_side + @first_side) > @second_side)
```
Comment (neb5562) [inline 3 - Rectangular triangle/solution_3.rb]: fixed long line @alexshvedko

File: 3 - Rectangular triangle/solution_3.rb [hunk 3]
```diff
@@ -32,7 +32,11 @@ def is_pythagorean?
 
   # check if triangle is isosceles && valid
   def is_isosceles?
-    @first_side == @second_side || @second_side == @third_side || @first_side == @third_side && is_triangle?
+    first_check = @first_side == @second_side
```
Comment (neb5562) [inline 3 - Rectangular triangle/solution_3.rb:35]: fixed long line in this method too @alexshvedko

File: 3 - Rectangular triangle/solution_3.rb (hunk 1 above)
Comment (neb5562) [reply 3 - Rectangular triangle/solution_3.rb]: @alexshvedko fixed

File: 3 - Rectangular triangle/solution_3.rb (hunk 3 above)
Comment (alexshvedko) [reply 3 - Rectangular triangle/solution_3.rb:35]: ok

**Review by alexshvedko — APPROVED** [review]

**Pull request merged**